CONTRIBUTING.md

```diff
@@ -2,26 +2,35 @@
 
 Thank you for considering contributing to this project! All contributors, big or small, are welcomed. To make the contribution process as smooth as possible, please follow the guidelines below.
 
+## Prerequisites
```

Comment:
These steps are missing to get started for a new developer. if you want I can create separate PR for this

Reply:
Nah, let's ship it with this PR. Thanks!